python/lsst/ts/MTAOS/model.py

```diff
@@ -352,14 +433,182 @@ async def run_wep(self, visit_id, extra_id, config, **kwargs):
             self.log.debug(
                 f"Processing MainCamera corner wavefront sensor on image {visit_id}."
             )
+            raise NotImplementedError(
+                "LSSTCam Corner wavefront sensor processing not implemented yet."
```

Comment:
Please update the doc string of this `NotImplementedError `. In addition, there is an empty line in your doc sting.